scripts/script.js

```diff
@@ -286,3 +286,35 @@ form.addEventListener('submit', (event) => {
     labelError.textContent = `${EMAIL_INVALID}!\n`;
   }
 });
+
+// end form validation
+
+// contact form local storage
+
+const inputElements = document.querySelectorAll('.contact-form-input');
+
+const contactMessage = {
+  firstname: '',
+  email: '',
+  message: '',
+};
+
+function populateStorage() {
+  localStorage.setItem('formData', JSON.stringify(contactMessage));
+}
+
+inputElements.forEach((input) => {
+  input.addEventListener('change', () => {
+    contactMessage[input.name] = form.elements[input.name].value.trim();
+    populateStorage();
+  });
+});
+
+window.addEventListener('load', () => {
+  const inputElements = document.querySelectorAll('.contact-form-input');
+  if (localStorage.getItem('formData') !== '' || localStorage.getItem('formData') !== null) {
+    inputElements.forEach((input) => {
+      input.value = JSON.parse(localStorage.getItem('formData'))[input.name];
+    });
+  }
+});
```

Comment:
- [ ] GOOD JOB :clap: :100: . I can see the contact form information is persistent to page reloads but only one is left. When the page is reloaded the input field(textarea) containing the message is not populated with the information from the `localstorage`. Kindly let's go ahead and fix it.  check here :point_down:  down what I can see when the page is loaded or reloaded
![Screenshot from 2021-12-17 11-34-27](https://user-images.githubusercontent.com/65068771/146524939-0dd6e31b-559f-469f-bea1-6685f7897193.png)

Reply:
Hello, 
Please find our demo here. It works correctly in our side and could you please take a look again. 
https://www.loom.com/share/d583db0be672434fa180cfbb68f121e6

![local-storage-snapshot](https://user-images.githubusercontent.com/85465559/146531245-25d82584-f2b6-45e4-ad33-96560326fff5.png)

Thank you.